schema/logs.go

```diff
@@ -0,0 +1,79 @@
+package schema
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+	"time"
+)
+
+// import or alias these from the provider?
+
+// LogRequest is the query to return the function logs.
+type LogRequest struct {
+	// Name is the function name and is required
+	Name string `json:"name"`
+	// Instance is the optional container name, that allows you to request logs from a specific function instance
+	Instance string `json:"instance"`
+	// Since is the optional datetime value to start the logs from
+	Since *time.Time `json:"since"`
+	// Limit sets the maximum number of log messages to return, <=0 means unlimited
+	Limit int `json:"limit"`
+	// Follow is allows the user to request a stream of logs
+	Follow bool `json:"follow"`
+	// Pattern is an optional regexp value to filter the log messages
+	Pattern *string `json:"pattern"`
+	// Invert allows you to control if the Pattern should be matched or negated
+	Invert bool `json:"invert"`
```

Comment:
Can this be changed to be more intuitive?

Pattern -> `Include`
Pattern + Invert  -> `Exclude`

or

Pattern -> `IncludePattern`
Pattern + Invert  -> `ExcludePattern`